dtreeviz/models/xgb_decision_tree.py

```diff
@@ -4,12 +4,18 @@
 from typing import List, Mapping
 
 import numpy as np
-import xgboost as xgb
-from xgboost.core import Booster
 
 from dtreeviz.models.shadow_decision_tree import VisualisationNotYetSupportedError
 from dtreeviz.models.shadow_decision_tree import ShadowDecTree
 
+try:
```

Comment:
if this is specific to xgb  we should be able to just import it right? the import itself is guarded in other files.

Reply:
idem